app/static/src/app/store/sensitivity/actions.ts

```diff
@@ -1,32 +1,78 @@
-import { ActionContext, ActionTree } from "vuex";
-import { AppState } from "../appState/state";
+import {
+    ActionContext, ActionTree, Commit, Dispatch
+} from "vuex";
+import { AppState, AppType } from "../appState/state";
 import { SensitivityState } from "./state";
 import { SensitivityGetter } from "./getters";
 import { SensitivityMutation } from "./mutations";
 import { RunAction } from "../run/actions";
 import userMessages from "../../userMessages";
 import { OdinSensitivityResult } from "../../types/wrapperTypes";
-import { BatchPars } from "../../types/responseTypes";
+import {
+    Batch, BatchPars, Odin, OdinRunnerDiscrete, OdinRunnerOde
+} from "../../types/responseTypes";
+import { ModelGetter } from "../model/getters";
 
 export enum SensitivityAction {
     RunSensitivity = "RunSensitivity",
-    RunSensitivityOnRehydrate = "RunSensitivityOnRehydrate"
+    RunSensitivityOnRehydrate = "RunSensitivityOnRehydrate",
+    ComputeNext = "ComputeNext"
 }
 
+const runModelIfRequired = (rootState: AppState, dispatch: Dispatch) => {
+    // Re-run model if required on sensitivity run so that plotted central traces are correct
+    if (rootState.run.runRequired) {
```

Comment:
Can we do this first? Otherwise the old central is confusingly seen on the plot.

To get a visually jarring version of this, toggle I0_at_steady_state between 0 and 1 running sensitivity